test/performance/short-running-tests/README.md

```diff
@@ -0,0 +1,213 @@
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```console
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) If using plain K8s clusters:
+
+   ```console
+   ./setup-clusterloader2.sh
+   ```
+
+   c) If using OpenShift clusters: 
+
+   ```console
+   ./setup-clusterloader2.sh --env ocp
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory
+
+   ```console
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   
+   ```console
+   vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaces to be created in your cluster to deploy the workload defined in step-2 above (*default value: 1*). For example: if `namespaces=2`, then the following namespaces will be created: `perf-exp-0` and `perf-exp-1`
+   - K8S_CLUSTER: set to `true` for plain Kubernetes clusters (*default value: "true"*)
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
+
+   To learn more about clusterloader2 tuningSet, see the following: [tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
+
+
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```console
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```console
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   In the following set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create your workload:
```

Comment:
"your cluster" is perhaps not entirely clear. I think you intend to refer to the WDS that clusterloader should thrash. If that is right then I think this could be worded more clearly.